fbw-a380x/src/base/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/model/behaviour/ecam-cp.xml

```diff
@@ -443,6 +445,16 @@
             <SIMVAR>A32NX_BTN_MORE</SIMVAR>
         </UseTemplate>
 
+        <!-- ECAM ALL -->
+        <UseTemplate Name="FBW_Anim_Interactions">
+            <ANIM_TYPE>BUTTON_HELD</ANIM_TYPE>
+            <ANIM_TEMPLATE>FBW_ECAM_BUTTON_Template</ANIM_TEMPLATE>
+            <BASE_NAME>ALL</BASE_NAME>
+            <PART_ID>ECAM_ALL</PART_ID>
+            <TOOLTIPID>Cycle through ECAM pages</TOOLTIPID>
```

Comment:
That seems to be true for the whole xml file - haven't found any of these keys in the locPak file. 

I agree it would be better to have it but I can"t put this effort into it right now.

Should happen in a separate PR for all controls.